apps/src/templates/sectionAssessments/assessmentsTestHelpers.js

```diff
@@ -7,16 +7,14 @@ export const studentOverviewData = [
     name: 'Caley',
     numMultipleChoiceCorrect: 7,
     numMultipleChoice: 10,
-    percentCorrect: '70%',
-    submissionStatus: 'Completed',
+    submissionStatus: true,
     submissionTimeStamp: '2/16/18 - 7:41 AM',
   },
   {
     id: 2,
     name: 'Maddie',
     numMultipleChoiceCorrect: 3,
     numMultipleChoice: 10,
-    percentCorrect: '',
     submissionStatus: 'In Progress',
```

Comment:
^`submissionStatus` should always be a boolean

Reply:
This submission status is invalid based on your proptype shapes. It should be `submissionStatus: false`, as well as on line 42.